### PR TITLE
Fix/apl 1404 open node health info without admin password

### PR DESCRIPTION
**Pull request by yuriy-larin-firstbridge**
- updated method /control/health to get node's health info _without admin password_
- added method /control/health-full to get full node's info **ONLY WITH admin password**
- added utility method /utils/setlog/level to set logging level dynamically on node

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java [hunk 1]
```diff
@@ -518,4 +523,53 @@ public Response hashByAlgorithm(
...
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = SetLogLevelDTO.class)))
+        }
+    )
+    @RolesAllowed("admin")
+    public Response setLoggingLevel(
+        @Parameter(name = "logLevel", description = "Valid log level from available list", required = true,
+            schema = @Schema(implementation = org.slf4j.event.Level.class))
+        @FormParam("logLevel") org.slf4j.event.Level logLevel,
+        @Parameter(description = "The full java package or logger name", required = true) @FormParam("packageName") String packageName,
```
Comment (AndrewBoyarsky) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java]: Move description to @Schema annotation to make it visible on swagger.

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java [hunk 2]
```diff
@@ -518,4 +523,53 @@ public Response hashByAlgorithm(
...
+            @ApiResponse(responseCode = "200", description = "Successful execution",
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = SetLogLevelDTO.class)))
+        }
+    )
+    @RolesAllowed("admin")
+    public Response setLoggingLevel(
+        @Parameter(name = "logLevel", description = "Valid log level from available list", required = true,
+            schema = @Schema(implementation = org.slf4j.event.Level.class))
+        @FormParam("logLevel") org.slf4j.event.Level logLevel,
```
Comment (AndrewBoyarsky) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java]: Better to apply javax.validation on parameters, i guess.

Comment (AndrewBoyarsky) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java]: org.slf3j.event.Level and ch.qos.logback.classic.Level both used here, i think if we use logback, then better to use level from logback package, isn't it?

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java (hunk 1 above)
Comment (yuriy-larin-firstbridge) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java]: Updated, but that doesn't show description

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java (hunk 2 above)
Comment (yuriy-larin-firstbridge) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java]: Added necessary

Comment (yuriy-larin-firstbridge) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java]: org.slf3j.event.Level suits much better for showing ENUM on web UI (as dropdown) then ch.qos.logback.classic.Level
That is reason for using two classes from both libraries/

**Pull request merged**